#DualShock4 JoyStick State message
float64 l_x
float64 l_y
float64 r_x
float64 r_y
